plugin-worldclock/lxqtworldclock.h

```diff
@@ -33,6 +33,7 @@
 
 #include <QDialog>
 #include <QLabel>
+#include <QToolTip>
```

Comment:
`#include <QToolTip>` can go into "lxqtworldclock.cpp"

Reply:
Yes, I noticed that nothing from this include is needed in *h file